### PR TITLE
add proper stateful example

### PR DESCRIPTION
**Pull request by flrnmrz**

### CODE REVIEW

File: go/driver/v2/driver.go [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+	if err != nil {
+		return err
+	}
+
+	data, err := json.Marshal(resp)
+	if err != nil {
+		return errors.New("failed to marshal in json decoded data")
+	}
+
+	fmt.Println(string(data))
```
Comment (bogh) [inline go/driver/v2/driver.go]: This print is required or is it for testing?

File: go/driver/v2/driver.go [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+	if err != nil {
+		return err
+	}
+
+	data, err := json.Marshal(resp)
+	if err != nil {
+		return errors.New("failed to marshal encode response to json")
+	}
+
+	fmt.Println(string(data))
```
Comment (bogh) [inline go/driver/v2/driver.go]: Another print.

File: go/driver/v2/driver.go (hunk 1 above)
Comment (flrnmrz) [reply go/driver/v2/driver.go]: It's required, afair. It's the primary output of the tool.

File: go/driver/v2/driver.go (hunk 2 above)
Comment (flrnmrz) [reply go/driver/v2/driver.go]: Same thing.

**Pull request merged**